internal/cfgplugins/staticnexthop.go

```diff
@@ -135,15 +137,31 @@ func NextHopGroupConfig(t *testing.T, dut *ondatra.DUTDevice, traffictype string
 	if deviations.NextHopGroupOCUnsupported(dut) {
 		switch dut.Vendor() {
 		case ondatra.ARISTA:
-			if traffictype == "v4" {
-				helpers.GnmiCLIConfig(t, dut, nextHopGroupConfigIPV4Arista)
-			} else if traffictype == "dualstack" {
-				helpers.GnmiCLIConfig(t, dut, nextHopGroupConfigDualStackIPV4Arista)
-				helpers.GnmiCLIConfig(t, dut, nextHopGroupConfigDualStackIPV6Arista)
-			} else if traffictype == "v6" {
-				helpers.GnmiCLIConfig(t, dut, nextHopGroupConfigIPV6Arista)
-			} else if traffictype == "multicloudv4" {
-				helpers.GnmiCLIConfig(t, dut, nextHopGroupConfigMulticloudIPV4Arista)
+			if params.StaticNHGName == "gre_encap" {
+				if traffictype == "dualstack" {
+					nextHopGreConfig := new(strings.Builder)
+					fmt.Fprintf(nextHopGreConfig, "nexthop-group %s type gre\n", params.StaticNHGName)
+					fmt.Fprint(nextHopGreConfig, "   ttl 64\n")
+					fmt.Fprint(nextHopGreConfig, "   fec hierarchical\n")
+
+					for i, nexthop := range params.NHIPAddrs {
+						fmt.Fprintf(nextHopGreConfig, "   entry  %d tunnel-destination %s tunnel-source %s\n", i, nexthop, fmt.Sprintf("10.235.143.%d", i))
```

Comment:
Don't nest fmt calls.  Create one template and supply as many args as needed.

```suggestion
						fmt.Fprintf(nextHopGreConfig, "   entry  %d tunnel-destination %s tunnel-source 10.235.143.%d\n", i, nexthop, i)
```